STATS/PlaystatsInstMissionEnd.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: STATS 
+---
+
+## _PLAYSTATS_INST_MISSION_END
+
+```c
+// 0xFEA3F7E83C0610FA 
+void _PLAYSTATS_INST_MISSION_END(Any p0);
+```
+
```

Comment:
This native activatesthe  following completion of instanced missions, commencing with the Fixer pack. This functionality isn't intended for competitive modes tho.